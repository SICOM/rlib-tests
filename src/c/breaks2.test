COMMAND=./breaks2
FORMATS="txt"
EXPECTED_pdf=
EXPECTED_xml=
EXPECTED_txt=
EXPECTED_csv=
EXPECTED_html=
