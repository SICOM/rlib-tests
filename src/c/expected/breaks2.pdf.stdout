%PDF-1.3
%·¾­ª
1 0 obj
<<
/Type /Catalog
/Pages 2 0 R
>>
endobj
2 0 obj
<<
/Type /Pages
/Kids [ 4 0 R ]
/Count 1
>>
endobj
3 0 obj
<<
/Producer (Haru Free PDF Library 2.2.1)
/Title (RLIB Report)
>>
endobj
4 0 obj
<<
/Type /Page
/MediaBox [ 0 0 612 792 ]
/Contents 5 0 R
/Resources <<
/ProcSet [ /PDF /Text /ImageB /ImageC /ImageI ]
/Font <<
/F1 7 0 R
/F2 8 0 R
>>
>>
/Parent 2 0 R
/Rotate 0
>>
endobj
5 0 obj
<<
/Length 6 0 R
>>
stream
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 11 Tf
/F2 11 Tf
BT
1 0 0 1 14.4 769.23999 Tm
(Breaks Example To Demonstrate Mixed Expressions With Variables) Tj
ET
/F1 11 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
BT
1 0 0 1 14.4 755.76001 Tm
(Company District Region Store      Sales  District share  Regional share   Company share) Tj
ET
0 0 0 rg
0 0 0 RG
14.4 752.59998 583.20001 0.856 re
f
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
BT
1 0 0 1 14.4 741.76001 Tm
(Foo, Inc.               ) Tj
ET
BT
1 0 0 1 144 741.76001 Tm
( ) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
14.4 737.59998 583.20001 1.856 re
f
BT
1 0 0 1 14.4 730.76001 Tm
( ) Tj
ET
BT
1 0 0 1 19.8 730.76001 Tm
(Region A               ) Tj
ET
BT
1 0 0 1 144 730.76001 Tm
( ) Tj
ET
BT
1 0 0 1 230.39999 730.76001 Tm
(                                 ) Tj
ET
14.4 726.59998 583.20001 1.856 re
f
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
BT
1 0 0 1 14.4 714.76001 Tm
(  ) Tj
ET
BT
1 0 0 1 25.2 714.76001 Tm
(District X            ) Tj
ET
BT
1 0 0 1 144 714.76001 Tm
( ) Tj
ET
BT
1 0 0 1 230.39999 714.76001 Tm
(                 ) Tj
ET
BT
1 0 0 1 403.20001 714.76001 Tm
( ) Tj
ET
14.4 711.59998 583.20001 0.856 re
f
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
BT
1 0 0 1 14.4 702.76001 Tm
(   ) Tj
ET
BT
1 0 0 1 30.6 702.76001 Tm
(Store \0431                  ) Tj
ET
BT
1 0 0 1 171 702.76001 Tm
( ) Tj
ET
BT
1 0 0 1 176.39999 702.76001 Tm
(  $1500.00) Tj
ET
BT
1 0 0 1 230.39999 702.76001 Tm
( ) Tj
ET
BT
1 0 0 1 316.79999 702.76001 Tm
( ) Tj
ET
BT
1 0 0 1 403.20001 702.76001 Tm
( ) Tj
ET
BT
1 0 0 1 489.60001 702.76001 Tm
( ) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
BT
1 0 0 1 14.4 693.76001 Tm
(   ) Tj
ET
BT
1 0 0 1 30.6 693.76001 Tm
(Store \0432                  ) Tj
ET
BT
1 0 0 1 171 693.76001 Tm
( ) Tj
ET
BT
1 0 0 1 176.39999 693.76001 Tm
(  $1300.00) Tj
ET
BT
1 0 0 1 230.39999 693.76001 Tm
( ) Tj
ET
BT
1 0 0 1 316.79999 693.76001 Tm
( ) Tj
ET
BT
1 0 0 1 403.20001 693.76001 Tm
( ) Tj
ET
BT
1 0 0 1 489.60001 693.76001 Tm
( ) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
14.4 688.59998 583.20001 0.856 re
f
BT
1 0 0 1 14.4 681.76001 Tm
(  ) Tj
ET
BT
1 0 0 1 25.2 681.76001 Tm
(District X            ) Tj
ET
BT
1 0 0 1 144 681.76001 Tm
( ) Tj
ET
BT
1 0 0 1 230.39999 681.76001 Tm
(                 ) Tj
ET
BT
1 0 0 1 403.20001 681.76001 Tm
( ) Tj
ET
14.4 678.59998 583.20001 0.856 re
f
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 149.39999 681.76001 Tm
(       $2800.00) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 235.8 693.76001 Tm
(          46.43) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 235.8 702.76001 Tm
(          53.57) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 149.39999 714.76001 Tm
(       $2800.00) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
BT
1 0 0 1 14.4 667.76001 Tm
(  ) Tj
ET
BT
1 0 0 1 25.2 667.76001 Tm
(District Y            ) Tj
ET
BT
1 0 0 1 144 667.76001 Tm
( ) Tj
ET
BT
1 0 0 1 230.39999 667.76001 Tm
(                 ) Tj
ET
BT
1 0 0 1 403.20001 667.76001 Tm
( ) Tj
ET
14.4 664.59998 583.20001 0.856 re
f
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
BT
1 0 0 1 14.4 655.76001 Tm
(   ) Tj
ET
BT
1 0 0 1 30.6 655.76001 Tm
(Store \0433                  ) Tj
ET
BT
1 0 0 1 171 655.76001 Tm
( ) Tj
ET
BT
1 0 0 1 176.39999 655.76001 Tm
(  $2000.00) Tj
ET
BT
1 0 0 1 230.39999 655.76001 Tm
( ) Tj
ET
BT
1 0 0 1 316.79999 655.76001 Tm
( ) Tj
ET
BT
1 0 0 1 403.20001 655.76001 Tm
( ) Tj
ET
BT
1 0 0 1 489.60001 655.76001 Tm
( ) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
BT
1 0 0 1 14.4 646.76001 Tm
(   ) Tj
ET
BT
1 0 0 1 30.6 646.76001 Tm
(Store \0434                  ) Tj
ET
BT
1 0 0 1 171 646.76001 Tm
( ) Tj
ET
BT
1 0 0 1 176.39999 646.76001 Tm
(  $1800.00) Tj
ET
BT
1 0 0 1 230.39999 646.76001 Tm
( ) Tj
ET
BT
1 0 0 1 316.79999 646.76001 Tm
( ) Tj
ET
BT
1 0 0 1 403.20001 646.76001 Tm
( ) Tj
ET
BT
1 0 0 1 489.60001 646.76001 Tm
( ) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
14.4 641.59998 583.20001 0.856 re
f
BT
1 0 0 1 14.4 634.76001 Tm
(  ) Tj
ET
BT
1 0 0 1 25.2 634.76001 Tm
(District Y            ) Tj
ET
BT
1 0 0 1 144 634.76001 Tm
( ) Tj
ET
BT
1 0 0 1 230.39999 634.76001 Tm
(                 ) Tj
ET
BT
1 0 0 1 403.20001 634.76001 Tm
( ) Tj
ET
14.4 631.59998 583.20001 0.856 re
f
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 149.39999 634.76001 Tm
(       $3800.00) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 235.8 646.76001 Tm
(          47.37) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 235.8 655.76001 Tm
(          52.63) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 149.39999 667.76001 Tm
(       $3800.00) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
BT
1 0 0 1 14.4 622.76001 Tm
( ) Tj
ET
BT
1 0 0 1 19.8 622.76001 Tm
(Region A               ) Tj
ET
BT
1 0 0 1 144 622.76001 Tm
( ) Tj
ET
BT
1 0 0 1 230.39999 622.76001 Tm
(                                 ) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 149.39999 622.76001 Tm
(       $6600.00) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 322.20001 634.76001 Tm
(          57.58) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 495 646.76001 Tm
(  4 \057  4) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 322.20001 646.76001 Tm
(          27.27) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 495 655.76001 Tm
(  3 \057  4) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 322.20001 655.76001 Tm
(          30.30) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 322.20001 667.76001 Tm
(          57.58) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 322.20001 681.76001 Tm
(          42.42) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 495 693.76001 Tm
(  2 \057  4) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 322.20001 693.76001 Tm
(          19.70) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 495 702.76001 Tm
(  1 \057  4) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 322.20001 702.76001 Tm
(          22.73) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 322.20001 714.76001 Tm
(          42.42) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 149.39999 730.76001 Tm
(       $6600.00) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
14.4 618.59998 583.20001 1.856 re
f
BT
1 0 0 1 14.4 611.76001 Tm
( ) Tj
ET
BT
1 0 0 1 19.8 611.76001 Tm
(Region B               ) Tj
ET
BT
1 0 0 1 144 611.76001 Tm
( ) Tj
ET
BT
1 0 0 1 230.39999 611.76001 Tm
(                                 ) Tj
ET
14.4 607.59998 583.20001 1.856 re
f
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
BT
1 0 0 1 14.4 595.76001 Tm
(  ) Tj
ET
BT
1 0 0 1 25.2 595.76001 Tm
(District M            ) Tj
ET
BT
1 0 0 1 144 595.76001 Tm
( ) Tj
ET
BT
1 0 0 1 230.39999 595.76001 Tm
(                 ) Tj
ET
BT
1 0 0 1 403.20001 595.76001 Tm
( ) Tj
ET
14.4 592.59998 583.20001 0.856 re
f
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
BT
1 0 0 1 14.4 583.76001 Tm
(   ) Tj
ET
BT
1 0 0 1 30.6 583.76001 Tm
(Store \04311                 ) Tj
ET
BT
1 0 0 1 171 583.76001 Tm
( ) Tj
ET
BT
1 0 0 1 176.39999 583.76001 Tm
(  $2500.00) Tj
ET
BT
1 0 0 1 230.39999 583.76001 Tm
( ) Tj
ET
BT
1 0 0 1 316.79999 583.76001 Tm
( ) Tj
ET
BT
1 0 0 1 403.20001 583.76001 Tm
( ) Tj
ET
BT
1 0 0 1 489.60001 583.76001 Tm
( ) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
BT
1 0 0 1 14.4 574.76001 Tm
(   ) Tj
ET
BT
1 0 0 1 30.6 574.76001 Tm
(Store \04312                 ) Tj
ET
BT
1 0 0 1 171 574.76001 Tm
( ) Tj
ET
BT
1 0 0 1 176.39999 574.76001 Tm
(  $2300.00) Tj
ET
BT
1 0 0 1 230.39999 574.76001 Tm
( ) Tj
ET
BT
1 0 0 1 316.79999 574.76001 Tm
( ) Tj
ET
BT
1 0 0 1 403.20001 574.76001 Tm
( ) Tj
ET
BT
1 0 0 1 489.60001 574.76001 Tm
( ) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
14.4 569.59998 583.20001 0.856 re
f
BT
1 0 0 1 14.4 562.76001 Tm
(  ) Tj
ET
BT
1 0 0 1 25.2 562.76001 Tm
(District M            ) Tj
ET
BT
1 0 0 1 144 562.76001 Tm
( ) Tj
ET
BT
1 0 0 1 230.39999 562.76001 Tm
(                 ) Tj
ET
BT
1 0 0 1 403.20001 562.76001 Tm
( ) Tj
ET
14.4 559.59998 583.20001 0.856 re
f
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 149.39999 562.76001 Tm
(       $4800.00) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 235.8 574.76001 Tm
(          47.92) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 235.8 583.76001 Tm
(          52.08) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 149.39999 595.76001 Tm
(       $4800.00) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
BT
1 0 0 1 14.4 548.76001 Tm
(  ) Tj
ET
BT
1 0 0 1 25.2 548.76001 Tm
(District N            ) Tj
ET
BT
1 0 0 1 144 548.76001 Tm
( ) Tj
ET
BT
1 0 0 1 230.39999 548.76001 Tm
(                 ) Tj
ET
BT
1 0 0 1 403.20001 548.76001 Tm
( ) Tj
ET
14.4 545.59998 583.20001 0.856 re
f
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
BT
1 0 0 1 14.4 536.76001 Tm
(   ) Tj
ET
BT
1 0 0 1 30.6 536.76001 Tm
(Store \04313                 ) Tj
ET
BT
1 0 0 1 171 536.76001 Tm
( ) Tj
ET
BT
1 0 0 1 176.39999 536.76001 Tm
(  $3000.00) Tj
ET
BT
1 0 0 1 230.39999 536.76001 Tm
( ) Tj
ET
BT
1 0 0 1 316.79999 536.76001 Tm
( ) Tj
ET
BT
1 0 0 1 403.20001 536.76001 Tm
( ) Tj
ET
BT
1 0 0 1 489.60001 536.76001 Tm
( ) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
BT
1 0 0 1 14.4 527.76001 Tm
(   ) Tj
ET
BT
1 0 0 1 30.6 527.76001 Tm
(Store \04314                 ) Tj
ET
BT
1 0 0 1 171 527.76001 Tm
( ) Tj
ET
BT
1 0 0 1 176.39999 527.76001 Tm
(  $2800.00) Tj
ET
BT
1 0 0 1 230.39999 527.76001 Tm
( ) Tj
ET
BT
1 0 0 1 316.79999 527.76001 Tm
( ) Tj
ET
BT
1 0 0 1 403.20001 527.76001 Tm
( ) Tj
ET
BT
1 0 0 1 489.60001 527.76001 Tm
( ) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
14.4 522.59998 583.20001 0.856 re
f
BT
1 0 0 1 14.4 515.76001 Tm
(  ) Tj
ET
BT
1 0 0 1 25.2 515.76001 Tm
(District N            ) Tj
ET
BT
1 0 0 1 144 515.76001 Tm
( ) Tj
ET
BT
1 0 0 1 230.39999 515.76001 Tm
(                 ) Tj
ET
BT
1 0 0 1 403.20001 515.76001 Tm
( ) Tj
ET
14.4 512.59998 583.20001 0.856 re
f
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 149.39999 515.76001 Tm
(       $5800.00) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 235.8 527.76001 Tm
(          48.28) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 235.8 536.76001 Tm
(          51.72) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 149.39999 548.76001 Tm
(       $5800.00) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
BT
1 0 0 1 14.4 503.76001 Tm
( ) Tj
ET
BT
1 0 0 1 19.8 503.76001 Tm
(Region B               ) Tj
ET
BT
1 0 0 1 144 503.76001 Tm
( ) Tj
ET
BT
1 0 0 1 230.39999 503.76001 Tm
(                                 ) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 149.39999 503.76001 Tm
(      $10600.00) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 322.20001 515.76001 Tm
(          54.72) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 495 527.76001 Tm
(  4 \057  4) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 322.20001 527.76001 Tm
(          26.42) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 495 536.76001 Tm
(  3 \057  4) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 322.20001 536.76001 Tm
(          28.30) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 322.20001 548.76001 Tm
(          54.72) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 322.20001 562.76001 Tm
(          45.28) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 495 574.76001 Tm
(  2 \057  4) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 322.20001 574.76001 Tm
(          21.70) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 495 583.76001 Tm
(  1 \057  4) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 322.20001 583.76001 Tm
(          23.58) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 322.20001 595.76001 Tm
(          45.28) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 149.39999 611.76001 Tm
(      $10600.00) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
14.4 497.60001 583.20001 1.856 re
f
BT
1 0 0 1 14.4 490.76001 Tm
(Foo, Inc.               ) Tj
ET
BT
1 0 0 1 144 490.76001 Tm
( ) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 149.39999 490.76001 Tm
(      $17200.00) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 408.60001 503.76001 Tm
(          61.63) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 408.60001 515.76001 Tm
(          33.72) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 408.60001 527.76001 Tm
(          16.28) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 408.60001 536.76001 Tm
(          17.44) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 408.60001 548.76001 Tm
(          33.72) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 408.60001 562.76001 Tm
(          27.91) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 408.60001 574.76001 Tm
(          13.37) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 408.60001 583.76001 Tm
(          14.53) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 408.60001 595.76001 Tm
(          27.91) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 408.60001 611.76001 Tm
(          61.63) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 408.60001 622.76001 Tm
(          38.37) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 408.60001 634.76001 Tm
(          22.09) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 408.60001 646.76001 Tm
(          10.47) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 408.60001 655.76001 Tm
(          11.63) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 408.60001 667.76001 Tm
(          22.09) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 408.60001 681.76001 Tm
(          16.28) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 408.60001 693.76001 Tm
(           7.56) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 408.60001 702.76001 Tm
(           8.72) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 408.60001 714.76001 Tm
(          16.28) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 408.60001 730.76001 Tm
(          38.37) Tj
ET
/F1 9 Tf
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 149.39999 741.76001 Tm
(      $17200.00) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf

endstream
endobj
6 0 obj
13699
endobj
7 0 obj
<<
/Type /Font
/BaseFont /Courier
/Subtype /Type1
/Encoding /WinAnsiEncoding
>>
endobj
8 0 obj
<<
/Type /Font
/BaseFont /Courier-Bold
/Subtype /Type1
/Encoding /WinAnsiEncoding
>>
endobj
xref
0 9
0000000000 65535 f
0000000015 00000 n
0000000064 00000 n
0000000123 00000 n
0000000205 00000 n
0000000401 00000 n
0000014154 00000 n
0000014175 00000 n
0000014270 00000 n
trailer
<<
/Root 1 0 R
/Info 3 0 R
/Size 9
>>
startxref
14370
%%EOF
